src/ui/buttons/props.js

```diff
@@ -163,6 +163,10 @@ export type Personalization = {|
             impression : string,
             click : string
         |}
+    |},
+    buttonAnimation : {|
```

Comment:
Should be optional and match other personalization types
`
buttonAnimation? : {|
        id : string,
        labelText : string,
        tracking : {|
            impression : string,
            click : string
        |}
    |}
    `